### DISCUSSION TITLE
【功能建议】哔哩哔哩漫画网页端解锁

### DISCUSSION THREAD

**Discussion by ZerOri** [the1812/Bilibili-Evolved#1328] [功能建议 / Ideas]
从这个月起，哔哩哔哩漫画无法在网页端阅读了，强制用户下载app，脚本有没有办法增加漫画区功能，解除这个限制？

![QQ截图20201217145703](https://user-images.githubusercontent.com/46775615/102454082-5fa78780-4078-11eb-91b6-f8d78e473d50.png)

**Comment by Geeyun-JY3** [comment]
我估计应该不会做

**Reply by the1812** [reply]
自信点, 不会做的
<img height="20" src="https://i0.hdslb.com/bfs/emote/bba7c12aa51fed0199c241465560dfc2714c593e.png">

**Reply by ZerOri** [reply]
> 自信点, 不会做的
> <img alt="" height="20" src="https://camo.githubusercontent.com/e67fae24a8f8f413dc9aa490ed575029c8b3a8440bce4ade92f820c78365d602/68747470733a2f2f69302e6864736c622e636f6d2f6266732f656d6f74652f626261376331326161353166656430313939633234313436353536306466633237313463353933652e706e67">

那就结束讨论吧
